AOC_Day1/ContentView.swift

```diff
@@ -14,53 +14,43 @@ struct ContentView: View {
     
     
     func rollingTotals(readings: [Int]) -> [Int] {
-        var returnArray: [Int] = []
+        var rollingTotals: [Int] = []
         for i in 0..<readings.count {
-            if i < 2 {
-                continue
-            } else {
-                returnArray.append( readings[i-2] + readings[i-1] + readings[i] )
+            if i >= 2 {
```

Comment:
The `if` branch with the `continue` seems a bit redundant here, if you flip the boolean `>= 2` rather than `< 2`, you can go straight into the code that you're trying to write.